sroka/api/athena/athena_api.py

```diff
@@ -17,42 +17,58 @@ def poll_status(session, _id):
         result = session.get_query_execution(
             QueryExecutionId=_id
         )
-    except ClientError:
-        print("Invalid query_id")
+    except ClientError as e:
+        if e.response['Error']['Code'] == 'InvalidRequestException':
+            print("Please check your query_id. Error message:")
+        else:
+            print("ClientError. Error message:")
+        print(e)
         return None
-    except EndpointConnectionError:
-        print('Please check your credentials including aws_region in config.ini file')
+    except EndpointConnectionError as e:
+        print('Please check your credentials including aws_region in config.ini file and Internet connection.',
+              'Error message:')
+        print(e)
         return None
 
     state = result['QueryExecution']['Status']['State']
 
-    if state == 'SUCCEEDED':
-        return result
-    elif state == 'FAILED':
+    if state in ('SUCCEEDED', 'FAILED', 'CANCELLED'):
         return result
     else:
+        print('Waiting for {} query to end'.format(_id))
         raise Exception
 
 
 def download_file(s3, s3_bucket, s3_key, filename):
     if filename:
         try:
             s3.Bucket(s3_bucket).download_file(s3_key, filename)
-        except FileNotFoundError:
-            print('File or folder not found.')
+        except FileNotFoundError as e:
+            print('File or folder not found. Error message:')
+            print(e)
             return None
-        except ClientError:
-            print('Please check your credentials including s3_bucket in config.ini file')
+        except ClientError as e:
+            if e.response['Error']['Code'] == 'InvalidRequestException':
+                print("Please check your query. Error message:")
+            elif e.response['Error']['Code'] == '':
+                print('')
```

Comment:
This is smelly. How can there be no error code? Shouldn't we print then "Unexpected error occurred"? 🙈

Reply:
Thank you, it was left there by mistake. Removed.